corehq/apps/integration/payments/services.py

```diff
@@ -118,7 +127,7 @@ def verify_payment_cases(domain, case_ids: list, verifying_user: WebUser):
     updated_cases = []
     for case_ids_chunk in chunked(case_ids, CHUNK_SIZE):
         cases_updates = _get_cases_updates(case_ids_chunk, payment_properties_update)
-        _, cases = handle_case_update(
+        xform, cases = handle_case_update(
```

Comment:
No need to change this, but by convention in the codebase we use `__` (double underscore) for unused variables so that it doesn't clash with `_`.